integrated_channels/utils.py

```diff
@@ -316,6 +316,18 @@ def generate_formatted_log(
         f'integrated_channel_plugin_configuration_id={plugin_configuration_id}, {message}'
 
 
+def log_exception(enterprise_configuration, msg, course_or_course_run_key=None):
+    LOGGER.exception(
+        generate_formatted_log(
+            channel_name=enterprise_configuration.channel_code(),
+            enterprise_customer_uuid=enterprise_configuration.enterprise_customer.uuid,
+            course_or_course_run_key=course_or_course_run_key,
+            plugin_configuration_id=enterprise_configuration.id,
+            message=msg
+        )
+    )
```

Comment:
@sameenfatima78 log_exception method is logging all of this information in the standard format, which is being passed through `enterprise_configuration`.